numpy/core/include/numpy/npy_math.h

```diff
@@ -258,8 +261,11 @@ float npy_copysignf(float x, float y);
 float npy_nextafterf(float x, float y);
 float npy_spacingf(float x);
 
+float npy_ldexpf(float x, int exp);
```

Comment:
npy_ldexp and npy_frexp in all three flavors already exist in master.  These came in as part of #4852, which pulled the ldexp and frexp changes from this.